docs/deploying_csi_vsphere_with_rbac.md

```diff
@@ -117,52 +120,114 @@ Create the secret by running the following command:
 [k8suser@k8master ~]$ kubectl create -f vcsi-secret.yaml
 ```
 
-#### 3. Create the RBAC roles
+#### 3. Create the RBAC roles and bindings for the CSI controller
 
-You can find the RBAC roles required by the provider in [csi-roles.yaml](https://github.com/kubernetes/cloud-provider-vsphere/raw/master/manifests/csi/csi-roles.yaml).
+You can find the RBAC roles and bindings required by the CSI controller in [vsphere-csi-controller-rbac.yaml](https://github.com/kubernetes/cloud-provider-vsphere/raw/master/manifests/csi/vsphere-csi-controller-rbac.yaml).
 
 To apply them to your Kubernetes cluster, run the following command:
 
 ```bash
-[k8suser@k8master ~]$ kubectl create -f csi-roles.yaml
+[k8suser@k8master ~]$ kubectl create -f vsphere-csi-controller-rbac.yaml
 ```
 
-#### 4. Create the RBAC role bindings
+#### 4. Create the RBAC roles and bindings for the CSI node
 
-You can find the RBAC role bindings required by the provider in [csi-role-bindings.yaml](https://github.com/kubernetes/cloud-provider-vsphere/raw/master/manifests/csi/csi-role-bindings.yaml).
+You can find the RBAC roles and bindings required by the CSI node manager in [vsphere-csi-node-rbac.yaml](https://github.com/kubernetes/cloud-provider-vsphere/raw/master/manifests/csi/vsphere-csi-node-rbac.yaml).
 
 To apply them to your Kubernetes cluster, run the following command:
 
 ```bash
-[k8suser@k8master ~]$ kubectl create -f csi-role-bindings.yaml
+[k8suser@k8master ~]$ kubectl create -f vsphere-csi-node-rbac.yaml
 ```
 
-#### 5. Deploy `csi-vsphere`
-
-You have two options for deploying `csi-vsphere`. It can be deployed either as a simple Pod or in a DaemonSet. There really isn't much difference between the two other than the DaemonSet will do leader election and the Pod will just assume to be the leader.
+#### 5. Deploy `csi-vsphere-controller`
 
 **IMPORTANT NOTES:**
-- Deploy either as a Pod or in a DaemonSet, but *DO NOT* deploy both.
-- The YAML to deploy as a Pod or a DaemonSet assume that your Kubernetes cluster was deployed using [kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/). If you deployed your cluster using alternate means, you will need to modify the either of the YAML files in order to provided necessary files or paths based on your deployment.
-
-##### Deploy `csi-vsphere` as a Pod
+- The YAML to deploy CSI controller assumes that your Kubernetes cluster was deployed using [kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/). If you deployed your cluster using alternate means, you will need to modify the either of the YAML files in order to provided necessary files or paths based on your deployment.
```

Comment:
> you will need to modify the either of the YAML files in order to provided necessary files or paths based on your deployment.

This may need some cleanup -- not sure what it is saying here.

Reply:
It basically means the default locations for various things like the kubernetes clusters certs and etc are where kubeadm placed them. If you rolled your cluster by hand or used some other alternate method, you need to update the YAML to reflect those locations.

Reply:
I was more picking on the grammar here, `the either of` and `to provided necessary` both are off...  So I would suggest:

> If you deployed your cluster using alternate means, you will need to modify the YAML files in order to provide necessary files or paths based on your deployment.